example/src/spa/view/showcase/showcase-view.js

```diff
@@ -0,0 +1,54 @@
+import './showcase.css';
+import cardsInfo from '../../../data/cards';
+import ElementBuilder from '../../../classes/util/element/element-builder';
+import View from '../../../classes/view/view';
+// eslint-disable-next-line no-unused-vars
+import Router from '../../router/router';
+import CardDetailView from '../card-detail/card-detail-view';
+import CardView from '../card/card-view';
+
+const CssClasses = {
+    SHOWCASE: 'showcase',
+};
+
+export default class ShowcaseView extends View {
+    /**
+     * @param {string} id
+     * @param {Router} router
+     */
+    constructor(router, id = '') {
+        super();
+        this.htmlElement = this.createView();
```

Comment:
`super({tag: 'section', classNames: [CssClasses.SHOWCASE]})` in a good structure